website_switzerland/models/event_registration.py

```diff
@@ -21,6 +23,12 @@ def create(self, vals_list):
                 registration.task_ids.filtered(
                     lambda t, m_task=child_protection_task: t.task_id == m_task
                 ).write({"done": True})
+
+            if not partner.country_id:
+                partner.country_id = 44
+
```

Comment:
I would prefer that we do this only for the specific event type. Other event types might have a different workflow. But the best would be a general behaviour that works for every event type. The method [_default_stage](https://github.com/compassionch/compassion-website/blob/51aac75c564499e9d68e14e7bb7a23f22c61a036/website_event_compassion/models/event_registration.py#L327) have this in mind but doesnt work for frontend submitted registrations, because the context is not set.
```suggestion
            if registration.event_id.event_type_id = self.env.ref("website_switzerland.event_type_group_visit"):
                registration.stage_id = self.env.ref("website_switzerland.stage_group_pay")
```